@re-/assert/test/assert.test.ts

```diff
@@ -1,12 +1,31 @@
 import { AssertionError, strict, throws } from "node:assert"
 import { join } from "node:path"
 import { dirName, readJson, writeJson } from "@re-/node"
+import { ExpressionStatement, SourceFile } from "ts-morph"
 import { assert } from "../src/index.js"
+import {
+    checkIfTestsContainErrors,
+    checkThatBenchSnapGetsPopulated,
+    cleanUpSourceFile,
+    getTestFileData
+} from "./metaTests/testHelpers.js"
 
 const n = 5
 const o = { re: "do" }
 const testDir = dirName()
 
+type TestData = {
+    statements: ExpressionStatement[]
+    expected: Record<number | string, string>
+    sourceFile: SourceFile
+    fullText: string
+    initialText: string
+}
+const snapShotMetaFile = "snapshotTests"
+const snapshotTemplate = "templateForSnapshots.ts"
```

Comment:
I like this approach of including the suffix as opposed to having to remember to do it in the utility function when you want to access the data. Let's update the other variables to include `.ts` as well.